app/components/ui/textarea.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react'
+
+import { cn } from '~/lib/utils'
+
+export interface TextareaProps
+  extends React.TextareaHTMLAttributes<HTMLTextAreaElement> {}
+
+const Textarea = React.forwardRef<HTMLTextAreaElement, TextareaProps>(
+  ({ className, ...props }, ref) => {
+    return (
```

Comment:
**nitpick**: Consider breaking down the className string for better readability.